spring-kafka-docs/src/main/asciidoc/appendix.adoc

```diff
@@ -56,37 +41,35 @@ When using 2.8.1, you also need to exclude `spring-kafka-test` 's `zookeeper` de
 
 <dependency>
     <groupId>org.apache.kafka</groupId>
-    <artifactId>kafka_2.13</artifactId>
-    <version>{kafka-version}</version>
+    <artifactId>kafka-clients</artifactId>
```

Comment:
Why move the clients artifact down here? Either way, it should be adjacent to the kafka-streams artifact.

Reply:
This one is `<classifier>test</classifier>`. I don't see any relevance to the `kafka-streams`.